### DISCUSSION TITLE
Mesh site credentials

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#1944] [General]
Hi guys!

What is the password of mesh? I tried all of possible (tactical, root, rmm user, etc.), but I can't do login.

![image](https://github.com/user-attachments/assets/9dbdb0f0-e6d5-4e95-a29d-bb2726f606a9)

**Comment by wh1te909** [comment]
The username/password were printed out at the end of the install script. If you didn't save it, you can reset it using the instructions here: https://docs.tacticalrmm.com/faq/#how-do-i-recover-my-meshcentral-login-credentials 

You can also run the following command to generate a url that will automatically log you in as the admin: https://docs.tacticalrmm.com/mesh_integration/#get-a-url-to-login-to-mesh-as-the-mesh-superuser

**Reply by ederpardeiro** [reply]
Thank you very much!!!
